ship-it-inc/ship-it-fend
# # fit build issue

#### What does this PR do?
Install the needed dependencies 

#### Description of Task to be completed?
- install the dependencies needed for production to work
#### How should this be manually tested?
- run npm run build
- check that the build was successful

#### Any background context you want to provide?
n/a
#### What are the relevant pivotal tracker stories?
n/a
#### Screenshots (if appropriate)
n/a
#### Questions:
n/a
